code/modules/food_and_drinks/food/foods/junkfood.dm

```diff
@@ -121,6 +121,16 @@
 	user.put_in_hands(L)
 	qdel(src)
 
+/obj/item/food/chocolate_bar
+	name = "Delux Chocolate-bar"
```

Comment:
delux is a name or you are going for deluxe?

Reply:
Typo